docs/_documentations/importing-existing-projects.md

```diff
@@ -21,17 +21,19 @@ Add a project from an existing Git repository:
 
 ## Requirements for importing an existing project not created in Codewind or through Appsody or Odo
 
-Local projects are built and run on Docker. To import your project to a local Codewind instance, you must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/).
-**Note:** The Dockerfile must contain an `EXPOSE` instruction to point to the port that the project runs on.
+Local projects are built and run on Docker. To import your project to a local Codewind instance, you must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/). **Note:** The Dockerfile must contain an `EXPOSE` instruction to point to the port that the project runs on.
 
 Remote projects are built and run on Kubernetes. To import your project to a remote Codewind instance, you must include a Dockerfile and a Helm chart. For more information about Dockerfiles and Helm charts, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/) and [Helm chart documentation](https://helm.sh/docs/topics/charts/).
 
+To import your own project into Codewind, follow these same requirements. Make sure to select the **Other (Basic Container)** option as the project type.
+
 ## What kind of projects can I add?
 
-Codewind develops cloud-native microservices. Therefore, each project must be self-sufficient, not reliant on other projects to build. 
+Codewind develops cloud-native microservices. Therefore, each project must be self-sufficient, not reliant on other projects to build. To determine the projects you want to develop, see the following:
 
-To import your own project into Codewind, follow the requirements outlined in [Requirements for importing an existing project not created in Codewind or through Appsody or Odo](#Requirements-for-importing-an-existing-project-not-created-in-Codewind-or-through-Appsody-or-Odo).
-Make sure to select the **Other (Basic Container)** option as the project type.
+* To simplify enhancement in containers and deploy cloud-native applications, work with [Codewind](https://www.eclipse.org/codewind/overview.html). 
```

Comment:
Should it be: work with Codewind-style projects?

Reply:
@j-c-berger I didn't see this comment addressed in the new commit?

Reply:
Yes, my bad, meant to tell you in a comment. I wanted to get your opinion on how ```develop a project with Codewind``` flowed. Sometimes I like to see several different wordings before I choose one. 

If you like ```Codewind-style projects``` more or if it makes more sense, I'll change it to that no problem :) 

Also, would we say ```odo-style projects```? Since it's a CLI tool, I wanted to make sure if that phrasing is still applicable. Thanks!

Reply:
`develop a project with Codewind` sounds fine on its own (I'm developing a project using "Codewind" as a tool), but that makes the other points `develop a project with Appsody/Odo` confusing IMO.  Because it sounds like for these latter cases, I'm **not** using Codewind anymore.

That's my thinking behind saying `work with Codewind/Appsody/ODO-style projects`, so it's still implied that we are using Codewind.  If there's a better way to convey that idea I'm all for it.

Reply:
Awesome, thanks, Andrew! Yes, we want to convey that they're using Codewind in all instances, but the project style varies pending on whether they use Appsody or odo. 

I'll get a new commit up shortly. I'm thinking your way of simply saying, ```Codewind/Appsody/odo-style``` is best, but maybe when I get in there, I'll think of something else. 

Thanks for the clarification!